##################################################
## Deprecated and will be removed in the future ##
##################################################

Header header

float64[3] hand_direction
float64[3] hand_normal
float64[3] hand_palm_pos
float64 hand_pitch
float64 hand_roll
float64 hand_yaw
